src/Elastic.Apm/Report/Serialization/LabelsJsonConverter.cs

```diff
@@ -11,7 +11,8 @@ public override void WriteJson(JsonWriter writer, Dictionary<string, string> lab
 			writer.WriteStartObject();
 			foreach (var keyValue in labels)
 			{
-				writer.WritePropertyName(SerializationUtils.TrimToPropertyMaxLength(keyValue.Key));
+				// Labels are trimmed and also de dotted in order to satisfy the Intake API
+				writer.WritePropertyName(SerializationUtils.TrimToPropertyMaxLength(keyValue.Key).Replace('.', '_'));
```

Comment:
We should also be replacing `*` and `"`, as they JSON Schema disallows those as well:

https://github.com/elastic/apm-server/blob/f633a79345d3333327d7de247fdf9901e0df2643/docs/spec/tags.json#L7

Reply:
Oh, right! Added those, thanks.